### PR TITLE
feat(transactions): add default sort order by created at

### LINKED ISSUE #309: Update transactions table
We should display the transactions in descending order - newest to oldest, by default.

Another good addition might be to show the date and time for each transaction, not only the date.
@Tymmmy thoughts on this? 👆

### PR DESCRIPTION
**Pull request by dragosp1011**
### Context
- fixes #309

### Changes

### CODE REVIEW

File: packages/wallet-backend/src/middlewares/validator.ts [hunk 1]
```diff
@@ -4,9 +4,10 @@ import { BadRequestException } from '../shared/models/errors/BadRequestException
 
 export async function zParse<T extends AnyZodObject>(
   schema: T,
-  req: Request
+  req: Request,
+  property: 'body' | 'query' = 'body'
```
Comment (raducristianpopa) [inline packages/wallet-backend/src/middlewares/validator.ts:8]:
```suggestion
  req: Request
```

File: packages/wallet-backend/src/middlewares/validator.ts [hunk 2]
```diff
@@ -4,9 +4,10 @@ import { BadRequestException } from '../shared/models/errors/BadRequestException
...
 ): Promise<z.infer<T>> {
-  const res = await schema.safeParseAsync(req.body)
+  const res = await schema.safeParseAsync(req[property])
```
Comment (raducristianpopa) [inline packages/wallet-backend/src/middlewares/validator.ts:10]:
```suggestion
  const res = await schema.safeParseAsync(req)
```

**Review by raducristianpopa — COMMENTED** [review]
LGTM. Looking more into it, I think we can improve the request validator.
Here is what I am thinking:
- pass the whole request object to the validator
- create validations schema for `body`, `query` and `params` in the same schema

The validation schema would look like this:
```ts
export const transactionListRequestSchema = z.object({
  body: z.object({ ... }), // if we expect a body
  query: z.object({ ... }), // if we expect query string parameters
  params: z.object({ ... }), // if we expect route parameters
})
```

What do you think about this?

**Review by Tymmmy — APPROVED** [review]

**Review by Tymmmy — APPROVED** [review]

File: packages/wallet-frontend/src/pages/account/[accountId]/[paymentPointerId].tsx [hunk 3]
```diff
@@ -142,7 +142,9 @@ export const getServerSideProps: GetServerSideProps<{
     createdAt: new Date(trx.createdAt).toLocaleDateString('default', {
       day: '2-digit',
       month: 'short',
-      year: 'numeric'
+      year: 'numeric',
+      hour: '2-digit',
+      minute: '2-digit'
```
Comment (raducristianpopa) [inline packages/wallet-frontend/src/pages/account/[accountId]/[paymentPointerId].tsx:147]: Should we relocate this transformation in the backend?

**Pull request merged**